frontends/p4/validateParsedProgram.h

```diff
@@ -88,7 +88,7 @@ class ValidateParsedProgram final : public Inspector {
     void postorder(const IR::P4Parser *parser) override {
         auto start = parser->states.getDeclaration(IR::ParserState::start);
         if (!start) {
-            ::P4::error(ErrorType::ERR_INVALID, "Parser %1% has no 'start' state", parser);
+            ::P4::error(ErrorType::ERR_INVALID, "Parser %1% has no 'start' state", parser->name);
```

Comment:
when changing this, please also remove the namespace, it is a relict

```suggestion
            error(ErrorType::ERR_INVALID, "Parser %1% has no 'start' state", parser->name);